### ISSUE TITLE
Rarity/invalid response when 'value' property in attributes is different than strings

### ISSUE THREAD

**Issue by kacper-koza-arianelabs** [hashgraph/hedera-nft-utilities#27]
### Description

The problem occurs when I use `calculateRarity`. I receive `'NaN'` as `totalRarity` when my attributes have `value` property different than string.

### Steps to reproduce

Place JSON files in a directory. Use `calculateRarity` with your directory location.

Directory content:
1.json 
```
{
  "name": "Example NFT 1",
  "image": "ipfs://QmRUQdj6tRoGhctPUNEnVPLKB94mtuikynHtwofbvfJVz9",
  "description": "This is an example NFT 1",
  "creator": "Hedera",
  "type": "image/png",
  "format": "HIP412@2.0.0",
  "attributes": [{ "trait_type": "color", "display_type": "color", "value": "rgb(255,0,0)" }],
  "properties": { "external_url": "https://nft.com/mycollection/1", "url": "https://nft.com/mycollection/1" }
}
```
2.json
```
{
  "name": "Example NFT 2",
  "image": "ipfs://QmQpAy1kFJfPuWPNABDzZZzZDuxqMjkSzxjnvmNvZ7KAny",
  "description": "This is an example NFT 2",
  "creator": "Hedera",
  "type": "image/png",
  "format": "HIP412@2.0.0",
  "attributes": [
    { "trait_type": "color", "display_type": "color", "value": "rgb(255,0,0)" },
    { "trait_type": "hasPipe", "display_type": "boolean", "value": true }
  ],
  "properties": { "external_url": "https://nft.com/mycollection/2", "url": "https://nft.com/mycollection/2" }
}
```
3.json
```
{
  "name": "Example NFT 3",
  "image": "ipfs://QmQdjadyPyETfvGtwn73sfJyZC1tFv89r9ZYARTQ14bBCF",
  "description": "This is an example NFT 3",
  "creator": "Hedera",
  "type": "image/png",
  "format": "HIP412@2.0.0",
  "attributes": [
    { "trait_type": "color", "display_type": "color", "value": "rgb(0,0,255)" },
    { "trait_type": "stamina", "display_type": "percentage", "value": 75 }
  ],
  "properties": { "external_url": "https://nft.com/mycollection/3", "url": "https://nft.com/mycollection/3" }
}
```
Result: 
```
[
  {
    attributeContributions: [ [Object] ],
    totalRarity: '1.00',
    NFT: 1,
    filename: '1.json'
  },
  {
    attributeContributions: [ [Object], [Object], [Object] ],
    totalRarity: 'NaN',
    NFT: 2,
    filename: '10.json'
  },
  {
    attributeContributions: [ [Object], [Object], [Object] ],
    totalRarity: 'NaN',
    NFT: 3,
    filename: '11.json'
  },
]
```

### Additional context

![image](https://github.com/hashgraph/hedera-nft-utilities/assets/129764375/c59e8710-f946-45cb-8fe4-35b9199605b3)

### Hedera network

mainnet, testnet, other

### Version

@hashgraph/nft-utilities  2.1.5

### Operating system

Windows

**Comment by michielmulders** [comment]
The HIP-412 specification describes that all regular values for attributes should be strings. The code does convert everything to strings. However, the bug here is that we are trying to calculate the rarity for all display types, which should be skipped in the calculation. I've opened a new PR that fixes this problem by not including them in the overall specification. These display types are usually used for games or aesthetics and don't influence overall rarity. 

This PR should fix the issue: https://github.com/hashgraph/hedera-nft-utilities/pull/28

**Comment by michielmulders** [comment]
Closed by #28

**Comment by michielmulders** [comment]
Closed by #28
